packages/reparse/reparse.2.0.0/opam

```diff
@@ -0,0 +1,39 @@
+opam-version: "2.0"
+synopsis: "Recursive descent parsing library for ocaml"
+description:
+  "Monadic, recursive descent based, parser construction library for ocaml. Comprehensively documented and tested."
+maintainer: ["Bikal Lem"]
+authors: ["Bikal Lem <gbikal@gmail.com>"]
+license: "MPL-2.0"
+homepage: "https://github.com/lemaetech/reparse"
+bug-reports: "https://github.com/lemaetech/reparse/issues"
+depends: [
+  "dune" {>= "2.7"}
+  "ocaml" {>= "4.10.0"}
+  "alcotest" {with-test}
+  "odoc" {with-doc}
+]
+build: [
+  ["dune" "subst"] {dev}
+  [
+    "dune"
+    "build"
+    "-p"
+    name
+    "-j"
+    jobs
+    "@install"
+    "@runtest" {with-test}
```

Comment:
Some tests for this package require `release-unix`. This would create a cyclic dependency, one solution could be to make these also tests in reparse-unix. Otherwise we can disable them, in the future opam 2.1 will be able to deal with this kind of cycles

Reply:
Could we disable the test, please. Thanks. The opam file is generated by dune. How do I disable tests in dune-project?

Reply:
Done.